server.js

```diff
@@ -20,4 +23,42 @@ app.get("/", function (request, response) {
   response.sendFile(__dirname + "/index.html");
 });
 
-app.listen(process.env.PORT);
+const getAllMessages = (req, res) => res.status(200).json({ data: messages });
+
+const postMessage = (req, res) => {
+  const { from, text } = req.body;
+
+  from.trim() === "" ? res.status(200).json("Fill from field please!") : null;
+  text.trim() === "" ? res.status(200).json("Fill text field please!") : null;
+  const newId = messages.length !== 0 ? messages[messages.length - 1].id + 1 : 0;
+  const newMessage = {
+    id: newId,
+    from: from,
+    text: text
+  }
+  messages.push(newMessage);
+  res.status(200).json("Your message added!");
```

Comment:
`res.status.json()` doesn't actually cause a return to happen - you need to manually add a `return` after each path. In this case, this would be better as simple `if` statements rather than ternarys (which should only be used to return one of 2 values from a functino):
```
const postMessage = (req, res) => {
   const {from, text} = req.body
   if (!from.trim()) {
      res.status(400).send('Please fill in `from`')
      return
  }
  if (!text.trim()) {...}
}
```